modules/EsetTools.py

```
@@ -99,9 +99,20 @@ def sendRequestForKey(self):
         elif self.mode == 'SMALL BUSINESS':
             uCE(self.driver, f"return {CLICK_WITH_BOOL}({GET_EBAV}('button', 'data-label', 'subscription-choose-trial-esbs-card-button'))")
         try:
-            exec_js(f'{GET_EBCN}("css-17v7x12")[0].click()')
+            for button in self.driver.find_elements('tag name', 'button'):
+                if button.get_attribute('innerText').strip().lower() == 'continue':
+                    exec_js(f'{GET_EBCN}("{button.get_attribute("class")}")[0].click()')
```

Ruff

`GET_EBCN` may be undefined, or defined from star imports

(F405)